testnet-contracts/migrations/2_next.js

```diff
@@ -1,7 +1,129 @@
-var Peggy = artifacts.require("Peggy");
-var TestToken = artifacts.require("TestToken");
+require("dotenv").config();
+
+// Deployment Summary
+// ====================================
+// Total deployments:   6
+// Final cost:          0.17950626 ETH
+
+const Valset = artifacts.require("Valset");
+const CosmosBridge = artifacts.require("CosmosBridge");
+const Oracle = artifacts.require("Oracle");
+const BridgeBank = artifacts.require("BridgeBank");
+const BridgeRegistry = artifacts.require("BridgeRegistry");
+const BridgeToken = artifacts.require("BridgeToken");
 
 module.exports = function(deployer, network, accounts) {
-  deployer.deploy(TestToken, { gas: 4612388, from: accounts[0] });
-  deployer.deploy(Peggy, { gas: 4612388, from: accounts[0] });
+  // Required initial paramater variables
+  let operator;
+  let initialValidators = [];
+  let initialPowers = [];
+
+  // Development network deployment param parsing/setting
+  if (network === "develop" || network === "ganache") {
+    // Operator
+    operator = accounts[0];
+    // Initial validators
+    const localValidatorCount = Number(process.env.LOCAL_VALIDATOR_COUNT);
+    if (localValidatorCount <= 0 || localValidatorCount > 9) {
+      return console.error(
+        "Must provide an initial validator count between 1-8 for local deployment."
+      );
+    } else {
+      initialValidators = accounts.slice(1, localValidatorCount + 1);
+    }
+    // Initial validator power
+    if (process.env.LOCAL_INITIAL_VALIDATOR_POWERS.length === 0) {
+      return console.error(
+        "Must provide initial local validator powers as environment variable."
+      );
+    } else {
+      initialPowers = process.env.LOCAL_INITIAL_VALIDATOR_POWERS.split(",");
+    }
+    // Testnet/mainnet network deployment param parsing/setting
+  } else {
```

Comment:
ditto

Reply:
Need this `else` for input validation on non-local networks, hopefully the updated formatting in https://github.com/cosmos/peggy/pull/82/commits/683353d328fd12d6c7c71c443f5a340ebea6dd69 makes this less confusing